### DISCUSSION TITLE
画质

### DISCUSSION THREAD

**Discussion by zzyhutouhunao** [Richasy/Bili.Uwp#1348] [常规]
以前的1080p和4k画质是没有了吗？

**Comment by Richasy** [comment]
非大会员不能看大会员清晰度，索性直接不显示了

**Comment by zzyhutouhunao** [comment]
知道了，问一下如果本身是大会员可以看

Richasy ***@***.***> 于2022年7月16日周六 22:50写道：

**Comment by zzyhutouhunao** [comment]
如果本身是大会员可以默认大会员画质吗，4k呢

Richasy ***@***.***> 于2022年7月16日周六 22:50写道：

**Reply by Richasy** [reply]
可以在设置/播放模式/画质偏好种选择画质优先

**Comment by zzyhutouhunao** [comment]
为什么微软商店昨天更新完今天又要更新，然后第二天还要更新，

Richasy ***@***.***> 于2022年7月21日周四 11:26写道：
